types/src/v0/traits.rs

```diff
@@ -615,6 +616,13 @@ impl<P: SequencerPersistence> Storage<SeqTypes> for Arc<P> {
     ) -> anyhow::Result<()> {
         (**self).append_quorum_proposal(proposal).await
     }
+
+    async fn update_decided_upgrade_certificate(
+        &self,
+        _decided_upgrade_certificate: Option<UpgradeCertificate<SeqTypes>>,
+    ) -> anyhow::Result<()> {
+        Ok(())
```

Comment:
I think we want to add `store_upgrade_certificate` to the `SequencerPersistence` trait then call that here